index.html

```diff
@@ -656,12 +656,23 @@ <h4>Amanda Jepson</h4>
 
         </div>
           <div class="row"> 
-            <!--new change-->
-            <!-- awesome code about me-->
-            <!--charlesmotaroki@students.uonbi.ac.ke	-->
-            <!--wanjaerica@gmail.com	-->
-            <!--mboyaivy@gmail.com-->
-            <!--machoguabednego6@gmail.com	-->
+            <div class="col-lg-3 col-md-6 d-flex align-items-stretch">
+              <div class="member" data-aos="fade-up" data-aos-delay="400">
+                <div class="member-img">
+                  <img src="assets/img/team/steve.jpeg" class="img-fluid" alt="">
+                  <div class="social">
+                    <a href=""><i class="bi bi-twitter"></i></a>
+                    <a href=""><i class="bi bi-facebook"></i></a>
+                    <a href=""><i class="bi bi-instagram"></i></a>
+                    <a href=""><i class="bi bi-linkedin"></i></a>
+                  </div>
+                </div>
+                <div class="member-info">
+                  <h4>Amanda Jepson</h4>
+                  <span>Accountant</span>

```

Comment:
There are no links to your profile and the name and job title is not related to you... you need to fill in this section with your details.